src/app/ui/learn/LearnPageArticlesSection/LearnPageArticlesFilteringBar.tsx

```
@@ -0,0 +1,77 @@
+  const { t } = useTranslation();
+  const { tab, tabs, changeTab } = useLearnFiltering();
+  const isTablet = useMediaQuery((theme) => theme.breakpoints.down('md'));
+
+  const tabOptions = useMemo(
+    () => [
+      {
+        value: TAG_ALL,
+        label: capitalizeString(TAG_ALL),
+        'data-testid': `blog-articles-tab-${TAG_ALL}`,
+      },
+      ...[...(tabs ?? [])]?.sort().map((tabLabel) => ({
```

Biome (2.4.7)

[error] Unsafe usage of optional chaining.

(lint/correctness/noUnsafeOptionalChaining)